ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -604,6 +604,39 @@ func (ipfs *Connector) RepoStat(ctx context.Context) (*api.IPFSRepoStat, error)
 	return &stats, nil
 }
 
+// RepoGC performs a garbage collection sweep on the repo.
+func (ipfs *Connector) RepoGC(ctx context.Context) ([]api.IPFSRepoGC, error) {
+	ctx, span := trace.StartSpan(ctx, "ipfsconn/ipfshttp/RepoGC")
+	defer span.End()
+
+	ctx, cancel := context.WithTimeout(ctx, ipfs.config.IPFSRequestTimeout)
+	defer cancel()
+
+	var repoGCResp []api.IPFSRepoGC
+	res, err := ipfs.doPostCtx(ctx, ipfs.client, ipfs.apiURL(), "repo/gc", "", nil)
+	if err != nil {
+		logger.Error(err)
+		return repoGCResp, err
+	}
+	defer res.Body.Close()
+
+	dec := json.NewDecoder(res.Body)
+	for {
+		resp := api.IPFSRepoGC{}
```

Comment:
cluster API and IPFS api will certainly use different types. This should be a type defined here that matches what the IPFS api returns, like every other method does in this module.